gui/controller/controller_docks.cpp

```diff
@@ -161,6 +161,8 @@ void Controller::setupDocks()
 			this, SLOT(unsubscribeFalseColor(QObject*,FalseColoring::Type)));
 	connect(falseColorDock, SIGNAL(falseColoringRecalcRequested(FalseColoring::Type)),
 			this, SLOT(recalcFalseColor(FalseColoring::Type)));
+	connect(falseColorDock->scaledView(), SIGNAL(pixelOverlay(int,int)),
```

Comment:
The controllers should not know all the widgets in the docks. this should be abstracted away. Please use a signal chain instead.